test/test-pmctl-local.js

```diff
@@ -57,6 +57,7 @@ helper.pmctl.waiton = waiton;
 function waiton(cmd, output) {
   while (true) {
     try {
+      console.trace('WAITON %s', cmd);
```

Comment:
If this is moved to the first line of `expect` then we can see non-`waiton` calls as well. For extra convenience, also log the expected value.